gradle/verification-metadata.xml

```diff
@@ -3092,9 +3107,14 @@
             <sha256 value="7a2bd04368bbf562ce2066015482a9801df08e6d776ad69f9e496bfad16cdb92" origin="Generated by Gradle"/>
          </artifact>
       </component>
+      <component group="org.jetbrains.kotlin.android" name="org.jetbrains.kotlin.android.gradle.plugin" version="1.8.20">
+         <artifact name="org.jetbrains.kotlin.android.gradle.plugin-1.8.20.pom">
+            <sha256 value="a805594a79703ae9fdaafbbf42c6cf00d57355213ce728b4c6716397ff949178" origin="Generated by Gradle"/>
+         </artifact>
+      </component>
       <component group="org.jetbrains.kotlin.plugin.serialization" name="org.jetbrains.kotlin.plugin.serialization.gradle.plugin" version="1.8.20">
          <artifact name="org.jetbrains.kotlin.plugin.serialization.gradle.plugin-1.8.20.pom">
-            <sha256 value="51e504aa44c7c572dfbc1ee08d7b447f22374511efbdbd8fabb46371187f1a1c" origin="Generated by Gradle"/>
+            <sha256 value="b81d0f71998ac7ad92315773746ee702d402e796432e0d360789d8f59923c762" origin="Generated by Gradle"/>
```

Comment:
Why has this hash changed?

Reply:
Really don't know why, so I've researched this, tried on different OSs and different machines. It seams if something in the code / dependencies is changed, the hash of this particular artifact is changing too while generating by Gradle. I've rebased pull request and this one from latest rebase / commit is reproduceable on all platforms.